test/acceptance/SeleniumSpec.scala

```diff
@@ -0,0 +1,456 @@
+      When("User hits checkout page ")
+      go.to(checkout)
+      assert(checkout.pageHasLoaded())
+
+      Then("the 'Product Payment' section should load")
+      assert(checkout.basketPreviewProductPaymentHasLoaded())
+
+      And("the payment amount and currency should be correct")
+      assert(checkout.basketPreviewProductPaymentContains("£37.50 every 3 months"))
+
+      And("the 'Pay in GBP' override should NOT load")
+      assert(!checkout.currencyOverrideHasLoaded())
+    }
+
+    scenario("Weekly quarterly sub purchase from EU for EU delivery", AcceptanceTest) {
+      checkDependenciesAreAvailable
+      val testUser = new TestUser
+      val checkout = Checkout(testUser, "checkout/weeklydomestic-gwoct18-quarterly-domestic?countryGroup=eu")
+      When("User hits checkout page ")
+      go.to(checkout)
+      assert(checkout.pageHasLoaded())
+
+      Then("the 'Product Payment' section should load")
+      assert(checkout.basketPreviewProductPaymentHasLoaded())
+
+      And("the payment amount and currency should be correct")
+      assert(checkout.basketPreviewProductPaymentContains("€61.30 every 3 months"))
+
+      And("the 'Pay in GBP' override should NOT load")
+      assert(!checkout.currencyOverrideHasLoaded())
+    }
+
+    scenario("Weekly quarterly sub purchase for ROW delivery", AcceptanceTest) {
+      checkDependenciesAreAvailable
+      val testUser = new TestUser
+      val checkout = Checkout(testUser, "checkout/weeklyrestofworld-gwoct18-quarterly-row?countryGroup=us")
+      When("User hits checkout page ")
+      go.to(checkout)
+      assert(checkout.pageHasLoaded())
+
+      Then("the 'Product Payment' section should load")
+      assert(checkout.basketPreviewProductPaymentHasLoaded())
+
+      And("the payment amount and currency should be correct")
+      assert(checkout.basketPreviewProductPaymentContains("US$81.30 every 3 months"))
+
+      And("the 'Pay in GBP' override should load")
+      assert(checkout.currencyOverrideHasLoaded())
+    }
+
+    scenario("Weekly quarterly sub purchase from ROW for UK delivery", AcceptanceTest) {
+      checkDependenciesAreAvailable
+      val testUser = new TestUser
+      val checkout = Checkout(testUser, "checkout/weeklydomestic-gwoct18-quarterly-domestic?countryGroup=uk")
+      When("User hits checkout page ")
+      go.to(checkout)
+      assert(checkout.pageHasLoaded())
+
+      Then("the 'Product Payment' section should load")
+      assert(checkout.basketPreviewProductPaymentHasLoaded())
+
+      And("the payment amount and currency should be correct")
+      assert(checkout.basketPreviewProductPaymentContains("£37.50 every 3 months"))
+
+      And("the 'Pay in GBP' override should NOT load")
+      assert(!checkout.currencyOverrideHasLoaded())
+    }
+  }
+
+  feature("Weekly Promotion Page") {
+
+      scenario("UK user lands on Guardian Weekly promotion page", AcceptanceTest) {
+        val promoCode = "/p/WWM99X"
+        val params = Some("gwoct18")
```

Comment:
I don't think we need the `gwoct18` param any more do we? Isn't that the default behaviour now?